### DISCUSSION TITLE
Access whatsapp api is blocked with codesandbox

### DISCUSSION THREAD

**Discussion by slim-hmidi** [codesandbox/codesandbox-client#6779] [Q&A]
I'm facing an issue while I type some text in the textField and I click on the button to send a text message on whatsapp but the page is blocked in codesandbox.
When I test it locally, the access to the url '`https://api.whatsapp.com/send?text=sometext' works fine.
Why codesandbox blocks the redirection?

Reproduction:
https://codesandbox.io/s/senddmwhatsapp-0tw96p?file=/src/App.js

**Comment by JamesACS** [comment]
Hey @slim-hmidi, this is a restriction of our in window preview only. If you pop the preview out into a new tab it will work!
